Tests/FluentAssertions.Equivalency.Specs/TestTypes.cs

```diff
@@ -410,6 +410,11 @@ public class Derived : Base
     public virtual string NonExcludedDerivedProperty => "Foo";
 }
 
+public class ClassWithArraySegment
```

Comment:
🔧 Please keep the test class just below the test. This `TestTypes.cs` is deprecated.

Reply:
Fixed.